Build failure: frigate

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

```text
nix build github:NixOS/nixpkgs/c3c698419f52e1e493abb160cf05feb3c42dec36#frigate
```

### Can Hydra reproduce this build failure?

Hydra is currently rebuilding this package.

### Link to Hydra build job

_No response_

### Relevant log output

```console
Sourcing python-remove-tests-dir-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing python-remove-bin-bytecode-hook.sh
Sourcing python-imports-check-hook.sh
Using pythonImportsCheckPhase
Sourcing python-namespaces-hook
Sourcing pytest-check-hook
Using pytestCheckPhase
Running phase: unpackPhase
unpacking source archive /nix/store/x9r38499iv7jgq2450hdmi23l79zkyxj-frigate-0.16.2-source
source root is frigate-0.16.2-source
setting SOURCE_DATE_EPOCH to timestamp 315619200 of file "frigate-0.16.2-source/web/vite.config.ts"
Running phase: patchPhase
applying patch /nix/store/c054sv8yjy2r16hjaa84f3m0vfbanbfk-constants.patch
patching file frigate/comms/webpush.py
applying patch /nix/store/6j7xj47jirmvkw073wpiqrih66i6asnf-b86e6e484f64bd43b64d7adebe78671a7a426edb.patch
patching file frigate/data_processing/common/face/model.py
Hunk #1 succeeded at 9 with fuzz 2.
Hunk #2 succeeded at 170 (offset -2 lines).
Hunk #3 succeeded at 280 (offset -2 lines).
applying patch /nix/store/xpjx9qi0c9xvil610yb9qyg0hb1cxr1l-ffmpeg.patch
patching file frigate/config/camera/ffmpeg.py
Hunk #2 succeeded at 71 (offset 1 line).
patching file frigate/record/export.py
Running phase: updateAutotoolsGnuConfigScriptsPhase
Running phase: configurePhase
no configure script, doing nothing
Running phase: installPhase
Running phase: fixupPhase
shrinking RPATHs of ELF executables and libraries in /nix/store/7wad9ah8saxvbd6sy57zzgkf8i38nb1l-frigate-0.16.2
checking for references to /build/ in /nix/store/7wad9ah8saxvbd6sy57zzgkf8i38nb1l-frigate-0.16.2...
patching script interpreter paths in /nix/store/7wad9ah8saxvbd6sy57zzgkf8i38nb1l-frigate-0.16.2
stripping (with command strip and flags -S -p) in  /nix/store/7wad9ah8saxvbd6sy57zzgkf8i38nb1l-frigate-0.16.2/lib
Executing pythonRemoveTestsDir
Finished executing pythonRemoveTestsDir
Running phase: installCheckPhase
no Makefile or custom installCheckPhase, doing nothing
Running phase: pythonCatchConflictsPhase
Running phase: pythonRemoveBinBytecodePhase
Running phase: pythonImportsCheckPhase
Executing pythonImportsCheckPhase
Running phase: pytestCheckPhase
Executing pytestCheckPhase
pytest flags: -m pytest -k not\ \(test_plus_labelmap\)
============================= test session starts ==============================
platform linux -- Python 3.13.8, pytest-8.4.2, pluggy-1.6.0
rootdir: /build/frigate-0.16.2-source
configfile: pyproject.toml
plugins: anyio-4.11.0, typeguard-4.4.4
collected 166 items / 1 deselected / 165 selected                              

frigate/test/http_api/test_http_app.py .                                 [  0%]
frigate/test/http_api/test_http_event.py ...........                     [  7%]
frigate/test/http_api/test_http_review.py .............................. [ 25%]
.                                                                        [ 26%]
frigate/test/test_birdseye.py .....                                      [ 29%]
frigate/test/test_camera_pw.py .....                                     [ 32%]
frigate/test/test_config.py ............................................ [ 58%]
........                                                                 [ 63%]
frigate/test/test_copy_yuv_to_position.py ...                            [ 65%]
frigate/test/test_ffmpeg_presets.py ..........                           [ 71%]
frigate/test/test_gpu_stats.py ..                                        [ 72%]
frigate/test/test_http.py ......F..                                      [ 78%]
frigate/test/test_obects.py .                                            [ 78%]
frigate/test/test_object_detector.py ....                                [ 81%]
frigate/test/test_record_retention.py ....                               [ 83%]
frigate/test/test_reduce_boxes.py ....                                   [ 86%]
frigate/test/test_storage.py ....                                        [ 88%]
frigate/test/test_video.py ................                              [ 98%]
frigate/test/test_yuv_region_2_rgb.py ...                                [100%]

=================================== FAILURES ===================================
___________________________ TestHttp.test_recordings ___________________________

self = <frigate.test.test_http.TestHttp testMethod=test_recordings>

    def test_recordings(self):
        app = create_fastapi_app(
            FrigateConfig(**self.minimal_config),
            self.db,
            None,
            None,
            None,
            None,
            None,
            None,
        )
        id = "123456.random"
    
        with TestClient(app) as client:
            _insert_mock_recording(id)
            response = client.get("/front_door/recordings")
            assert response.status_code == 200
            recording = response.json()
>           assert recording
E           assert []

frigate/test/test_http.py:355: AssertionError
------------------------------ Captured log call -------------------------------
WARNING  frigate.util.services:services.py:595 Did not detect hwaccel, using a GPU for accelerated video decoding is highly recommended
=============================== warnings summary ===============================
../../nix/store/zmj48kwjn41rkny6863nrxgv1inmm6i8-python3.13-filterpy-1.4.5-unstable-2022-08-23/lib/python3.13/site-packages/filterpy/discrete_bayes/discrete_bayes.py:22
  /nix/store/zmj48kwjn41rkny6863nrxgv1inmm6i8-python3.13-filterpy-1.4.5-unstable-2022-08-23/lib/python3.13/site-packages/filterpy/discrete_bayes/discrete_bayes.py:22: DeprecationWarning: Please import `convolve` from the `scipy.ndimage` namespace; the `scipy.ndimage.filters` namespace is deprecated and will be removed in SciPy 2.0.0.
    from scipy.ndimage.filters import convolve

../../nix/store/zmj48kwjn41rkny6863nrxgv1inmm6i8-python3.13-filterpy-1.4.5-unstable-2022-08-23/lib/python3.13/site-packages/filterpy/discrete_bayes/discrete_bayes.py:23
  /nix/store/zmj48kwjn41rkny6863nrxgv1inmm6i8-python3.13-filterpy-1.4.5-unstable-2022-08-23/lib/python3.13/site-packages/filterpy/discrete_bayes/discrete_bayes.py:23: DeprecationWarning: Please import `shift` from the `scipy.ndimage` namespace; the `scipy.ndimage.interpolation` namespace is deprecated and will be removed in SciPy 2.0.0.
    from scipy.ndimage.interpolation import shift

frigate/test/test_camera_pw.py:12
  /build/frigate-0.16.2-source/frigate/test/test_camera_pw.py:12: SyntaxWarning: invalid escape sequence '\{'
    "rtsp://user:password`~!@#$%^&*()-_;',.<>:\"\{\}\[\]@@192.168.0.2:554/live"

frigate/test/http_api/test_http_app.py: 30 warnings
frigate/test/http_api/test_http_event.py: 330 warnings
frigate/test/http_api/test_http_review.py: 930 warnings
frigate/test/test_http.py: 270 warnings
frigate/test/test_storage.py: 120 warnings
  /nix/store/gmvv0150a9y1q06163yldhancsp1gp4x-python3.13-peewee-3.18.2/lib/python3.13/site-packages/peewee.py:6439: DeprecationWarning: datetime.datetime.utcnow() is deprecated and scheduled for removal in a future version. Use timezone-aware objects to represent datetimes in UTC: datetime.datetime.now(datetime.UTC).
    dd[field_name] = default()

frigate/test/http_api/test_http_app.py: 1 warning
frigate/test/http_api/test_http_event.py: 11 warnings
frigate/test/http_api/test_http_review.py: 31 warnings
frigate/test/test_http.py: 9 warnings
frigate/test/test_storage.py: 4 warnings
  <string>:7: DeprecationWarning: change_columns is depricated. Use change_fields instead.

frigate/test/http_api/test_http_app.py: 1 warning
frigate/test/http_api/test_http_event.py: 11 warnings
frigate/test/http_api/test_http_review.py: 31 warnings
frigate/test/test_http.py: 9 warnings
frigate/test/test_storage.py: 4 warnings
  <string>:77: DeprecationWarning: python is depricated. Use run instead.

frigate/test/http_api/test_http_app.py: 2 warnings
frigate/test/http_api/test_http_event.py: 11 warnings
frigate/test/http_api/test_http_review.py: 31 warnings
frigate/test/test_http.py: 9 warnings
  /build/frigate-0.16.2-source/frigate/api/fastapi_app.py:96: DeprecationWarning: 
          on_event is deprecated, use lifespan event handlers instead.
  
          Read more about it in the
          [FastAPI docs for Lifespan Events](https://fastapi.tiangolo.com/advanced/events/).
          
    @app.on_event("startup")

frigate/test/http_api/test_http_app.py: 2 warnings
frigate/test/http_api/test_http_event.py: 11 warnings
frigate/test/http_api/test_http_review.py: 31 warnings
frigate/test/test_http.py: 9 warnings
  /nix/store/caj71axadd9c9a8d2dw84dw8ddwcfjg7-python3.13-fastapi-0.116.1/lib/python3.13/site-packages/fastapi/applications.py:4495: DeprecationWarning: 
          on_event is deprecated, use lifespan event handlers instead.
  
          Read more about it in the
          [FastAPI docs for Lifespan Events](https://fastapi.tiangolo.com/advanced/events/).
          
    return self.router.on_event(event_type)

frigate/test/http_api/test_http_event.py: 16 warnings
frigate/test/test_http.py: 3 warnings
  /build/frigate-0.16.2-source/frigate/api/event.py:95: DeprecationWarning: deprecated
    include_thumbnails = params.include_thumbnails

frigate/test/test_object_detector.py::TestLocalObjectDetector::test_detect_given_tensor_input_should_return_lfiltered_detections
  /build/frigate-0.16.2-source/frigate/test/test_object_detector.py:119: PydanticDeprecatedSince20: `parse_obj_as` is deprecated. Use `pydantic.TypeAdapter.validate_python` instead. Deprecated in Pydantic V2.0 to be removed in V3.0. See Pydantic V2 Migration Guide at https://errors.pydantic.dev/2.11/migration/
    test_cfg = parse_obj_as(DetectorConfig, {"type": "cpu", "model": {}})

frigate/test/test_object_detector.py::TestLocalObjectDetector::test_detect_raw_given_tensor_input_should_call_api_detect_raw_with_transposed_tensor
  /build/frigate-0.16.2-source/frigate/test/test_object_detector.py:70: PydanticDeprecatedSince20: `parse_obj_as` is deprecated. Use `pydantic.TypeAdapter.validate_python` instead. Deprecated in Pydantic V2.0 to be removed in V3.0. See Pydantic V2 Migration Guide at https://errors.pydantic.dev/2.11/migration/
    test_cfg = parse_obj_as(DetectorConfig, {"type": "cpu", "model": {}})

frigate/test/test_object_detector.py::TestLocalObjectDetector::test_detect_raw_given_tensor_input_should_return_api_detect_raw_result
  /build/frigate-0.16.2-source/frigate/test/test_object_detector.py:47: PydanticDeprecatedSince20: `parse_obj_as` is deprecated. Use `pydantic.TypeAdapter.validate_python` instead. Deprecated in Pydantic V2.0 to be removed in V3.0. See Pydantic V2 Migration Guide at https://errors.pydantic.dev/2.11/migration/
    detector_config=parse_obj_as(DetectorConfig, {"type": "cpu", "model": {}})

frigate/test/test_object_detector.py::TestLocalObjectDetector::test_localdetectorprocess_should_only_create_specified_detector_type
frigate/test/test_object_detector.py::TestLocalObjectDetector::test_localdetectorprocess_should_only_create_specified_detector_type
frigate/test/test_object_detector.py::TestLocalObjectDetector::test_localdetectorprocess_should_only_create_specified_detector_type
frigate/test/test_object_detector.py::TestLocalObjectDetector::test_localdetectorprocess_should_only_create_specified_detector_type
frigate/test/test_object_detector.py::TestLocalObjectDetector::test_localdetectorprocess_should_only_create_specified_detector_type
frigate/test/test_object_detector.py::TestLocalObjectDetector::test_localdetectorprocess_should_only_create_specified_detector_type
frigate/test/test_object_detector.py::TestLocalObjectDetector::test_localdetectorprocess_should_only_create_specified_detector_type
frigate/test/test_object_detector.py::TestLocalObjectDetector::test_localdetectorprocess_should_only_create_specified_detector_type
  /build/frigate-0.16.2-source/frigate/test/test_object_detector.py:22: PydanticDeprecatedSince20: `parse_obj_as` is deprecated. Use `pydantic.TypeAdapter.validate_python` instead. Deprecated in Pydantic V2.0 to be removed in V3.0. See Pydantic V2 Migration Guide at https://errors.pydantic.dev/2.11/migration/
    test_cfg = parse_obj_as(

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED frigate/test/test_http.py::TestHttp::test_recordings - assert []
==== 1 failed, 164 passed, 1 deselected, 1931 warnings in 83.56s (0:01:23) =====
```

### Additional context

Disabling `test_recordings` allows frigate to build successfully.

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.17.4, NixOS, 25.11 (Xantusia), 25.11.20251023.87f5d25`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.31.2`
 - nixpkgs: `/nix/store/mkvig80qmirl3xqj5ar8cdcqdil9w97f-source`



### Notify maintainers

@mweinelt

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/op/openvino/package.nix
+++ b/pkgs/by-name/op/openvino/package.nix
@@ -118,7 +118,7 @@ stdenv.mkDerivation rec {
     "-DCMAKE_MODULE_PATH:PATH=${placeholder "out"}/lib/cmake"
     "-DCMAKE_PREFIX_PATH:PATH=${placeholder "out"}"
     "-DOpenCV_DIR=${lib.getLib opencv}/lib/cmake/opencv4/"
-    "-DCMAKE_PREFIX_PATH=${lib.getLib protobuf}"
+    "-DProtobuf_LIBRARIES=${protobuf}/lib/libprotobuf${stdenv.hostPlatform.extensions.sharedLibrary}"
     "-DPython_EXECUTABLE=${python.interpreter}"
 
     (cmakeBool "CMAKE_VERBOSE_MAKEFILE" true)
@@ -142,8 +142,7 @@ stdenv.mkDerivation rec {
     # system libs
     (cmakeBool "ENABLE_SYSTEM_FLATBUFFERS" true)
     (cmakeBool "ENABLE_SYSTEM_OPENCL" true)
-    (cmakeBool "ENABLE_SYSTEM_PROTOBUF" true)
-    (cmakeBool "Protobuf_USE_STATIC_LIBS" false)
+    (cmakeBool "ENABLE_SYSTEM_PROTOBUF" false)
     (cmakeBool "ENABLE_SYSTEM_PUGIXML" true)
     (cmakeBool "ENABLE_SYSTEM_SNAPPY" true)
     (cmakeBool "ENABLE_SYSTEM_TBB" true)
@@ -164,7 +163,6 @@ stdenv.mkDerivation rec {
     libxml2
     ocl-icd
     opencv
-    protobuf
     pugixml
     snappy
     onetbb

